### PR TITLE
Implementation of POM

### PR DESCRIPTION
**Pull request by danielopez26**

### CODE REVIEW

**Comment by danielopez26** [comment]
I am returning a boolean as my validation but I have a doubt about it, where do I put my assert? in a Test class?

So far, I'm working to simplify the code even more.

**Comment by danielopez26** [comment]
I have modified my test, I get a string from the payment class and I validate it in my test

File: src/main/java/pagesObject/SignInPage.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    @FindBy(id = "email")
+    private WebElement email;
+
+    @FindBy(id = "passwd")
+    private WebElement password;
+
+    @FindBy(id = "SubmitLogin")
+    private WebElement btnsignIn;
+
+    public void login (){
```
Comment (aperdomob) [inline src/main/java/pagesObject/SignInPage.java:30]: the user and password should be input parameters

File: src/test/java/TestAutomationPractice.java [hunk 2]
```diff
@@ -3,76 +3,43 @@
...
-    public WebDriver driver = new Drivers().strategyDriver();
-    WebDriverWait wait =  new WebDriverWait(driver,20);
+    WebDriver driver = new Drivers().strategyDriver();
+    HomePage homePage = new HomePage(driver);
+    CartPage cartPage = new CartPage(driver);
+    SummaryPage summaryPage =  new SummaryPage(driver);
+    SignInPage signInPage = new SignInPage(driver);
+    AddressPage addressPage = new AddressPage(driver);
+    ShippingPage shippingPage =  new ShippingPage(driver);
+    PaymentPage paymentPage = new PaymentPage(driver);
```
Comment (aperdomob) [inline src/test/java/TestAutomationPractice.java:19]: instance all variables into before

File: src/test/java/TestAutomationPractice.java [hunk 3]
```diff
@@ -3,76 +3,43 @@
...
-        WebElement payByBankWire = driver.findElement(By.className("bankwire"));
-        payByBankWire.click();
-
-        WebElement confirmOrder = driver.findElement(By.xpath("//button[span = \"I confirm my order\"]"));
-        confirmOrder.click();
-
-        WebElement orderConfirmationMessage = driver.findElement(By.xpath("//strong[text() = \"Your order on My Store is complete.\"]"));
-        Assertions.assertEquals(orderConfirmationMessage.getText(), "Your order on My Store is complete.");
-    }
+        homePage.addToCart();
```
Comment (aperdomob) [inline src/test/java/TestAutomationPractice.java:30]: que estas agregando, sea más explicito en el nombre del metodo para que saber si agregas un vestion, pantalon, blusa, etc

**Review by aperdomob — CHANGES_REQUESTED** [review]
recuerde agregarle private a todos las variables de las clases

File: src/test/java/TestAutomationPractice.java [hunk 4]
```diff
@@ -3,76 +3,50 @@
...
-    public WebDriver driver = new Drivers().strategyDriver();
-    WebDriverWait wait =  new WebDriverWait(driver,20);
+    WebDriver driver = new Drivers().strategyDriver();
+    HomePage homePage;
+    CartPage cartPage;
+    SummaryPage summaryPage;
+    SignInPage signInPage;
+    AddressPage addressPage;
+    ShippingPage shippingPage;
+    PaymentPage paymentPage;
```
Comment (aperdomob) [inline src/test/java/TestAutomationPractice.java:19]: add private

**Review by aperdomob — CHANGES_REQUESTED** [review]

**Comment by danielopez26** [comment]
Added private

**Review by aperdomob — APPROVED** [review]

**Pull request merged**